### PR TITLE
Rename module (WMI -> Windows)

### PR DESCRIPTION
**Pull request by thiagoftsm**
Fixes https://github.com/netdata/netdata/issues/14439

This PR is modifying `wmi` collector to `windows monitoring`.

Tests were already done in one of our VMs where charts appear with new 
[contexts](https://github.com/netdata/go.d.plugin/files/10691812/charts.txt):

Screenshot_20230209_013810

### CODE REVIEW

File: modules/windows/collect.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ const (
 	collectorNetFrameworkCLRSecurity        = "netframework_clrsecurity"
 )
 
-func (w *WMI) collect() (map[string]int64, error) {
+func (w *WINDOWS) collect() (map[string]int64, error) {
```
Comment (ilyam8) [inline modules/windows/collect.go:38]: Should be just `Windows`

**Review by ilyam8 — CHANGES_REQUESTED** [review]

File: modules/windows/collect.go (hunk 1 above)
Comment (thiagoftsm) [reply modules/windows/collect.go:38]: I updated documentation removing `monitoring`.

**Comment by thiagoftsm** [comment]
I convert to draft while we discuss compatibility break.

File: modules/windows/collect.go (hunk 1 above)
Comment (ilyam8) [reply modules/windows/collect.go:38]: I didn't mean documentation: `func (w *WINDOWS)` => `func (w *Windows)`

Comment (thiagoftsm) [reply modules/windows/collect.go]: I pushed another commit renaming it. I was using the same pattern we have in `netdata/netdata`.

**Pull request merged**